static/src/javascripts/projects/common/modules/analytics/shouldCaptureMetrics.ts

```diff
@@ -2,12 +2,14 @@ import type { ABTest } from '@guardian/ab-core';
 import { getUrlVars } from 'lib/url';
 import { isInABTestSynchronous } from '../experiments/ab';
 import { integrateIma } from '../experiments/tests/integrate-ima';
+import { noCarrotAdsNearNewsletterSignupBlocks } from '../experiments/tests/no-carrot-ads-near-newsletter-signup-blocks';
 import { removePrebidA9Canada } from '../experiments/tests/removePrebidA9Canada';
 
 const defaultClientSideTests: ABTest[] = [
 	/* linter, please keep this array multi-line */
 	integrateIma,
 	removePrebidA9Canada,
+	noCarrotAdsNearNewsletterSignupBlocks,
```

Comment:
@chrislomaxjones is this doing a similar thing as the `bypassMetricsSampling` from the teads liveblog test?

Reply:
I think this is the way to do it before [this PR](https://github.com/guardian/frontend/pull/25121) introduced a new method. I think it probably still works, but worth checking @simonbyford - otherwise the new method is quite succinct.

Reply:
Ahhh very neat, thanks for the heads up! I have amended it now